contracts/pallet-ibc/src/events.rs

```diff
@@ -433,6 +438,10 @@ impl From<RawIbcEvent> for IbcEvent {
 				kind: ev.kind.as_bytes().to_vec(),
 				module_id: ev.module_name.to_string().as_bytes().to_vec(),
 			},
+			RawIbcEvent::PushWasmCode(ev) => {
```

Comment:
I think we should remove this piece. As far as I understand this functionality will be added on the cosmos chain itself, and the user can run it from the CLI. Is it because it's useful to run the interchaintests?

Reply:
Since it's a core IBC functionality, I think it's better to have it, although parachain doesn't support it yet